pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "azure-operator"
 	source      string = "https://github.com/giantswarm/azure-operator"
-	version            = "4.1.0-dev"
+	version            = "4.1.0-nikola-flowcharts"
```

Comment:
Better remove this from the PR

Reply:
Yeah, wanted to remove it, thanks for the reminder :)